mail/api/mail.py

```diff
@@ -159,16 +156,49 @@ def group_recipients_and_add_attachments(rows: list[dict]) -> list[dict]:
 			parent = attachment.pop("parent")
 			attachments_by_parent[parent].append(attachment)
 
+		else:
```

Comment:
```suggestion
		elif attachment.disposition == "inline":
```

disposition can be `None`